lib/licensee/projects/uri_project.rb

```diff
@@ -0,0 +1,25 @@
+# URI-based project
+#
+# Analyzes a given URI for license information
+require 'net/http'
+require 'uri'
+
+module Licensee
+  class UriProject < Project
+    def initialize(path, **args)
```

Comment:
I think we'll want to raise an `UnsuportedProeject` error here if `allow_remote` is false, rather than raising when `load_file` is first called. This matches the behavior of FSProject, and will allow us to add an additional project type in the last after URIProject if we want down the line. This will also allow us to get ride of the `@allow_remote` guards on each method.